easybuild/easyconfigs/c/CuPy/CuPy-11.4.0-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'CuPy'
+version = '11.4.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://cupy.dev'
+description = "CuPy is an open-source array library accelerated with NVIDIA CUDA."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+builddependencies = [
+    ('pytest', '7.2.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('cuDNN', '8.2.2.26', versionsuffix, SYSTEM),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('cuTENSOR', '1.6.1.5', versionsuffix, SYSTEM),
+    # ('cuSPARSELt', '0.3.0.3', versionsuffix, SYSTEM),
```

Comment:
Why is this commented? If it's really not needed, can't it just be removed?

Reply:
It was an optional thing, but i had problems getting that to work in my PR. I've forgotten what the reason was, only that i didn't care enough about that optional feature.

Reply:
It currently crashes/fails to compile some part, I left in in place to be enabled for a newer toolchain

Reply:
Should we put that as a comment? I.e. that it's an optional dep, commented out because it gave issues, but could be re-enabled for newer toolchain?